CONTRIBUTING.md

```diff
@@ -113,7 +113,159 @@ You can use `git rebase --interactive` to reorganize your commits and  `git
 push --force`  to update your branch. Depending on your IDE, there are various
 features or plugins to streamline this process (e.g.,
 [vim-fugitive](https://github.com/tpope/vim-fugitive),
-[magit](https://magit.vc/)).
+[magit](https://magit.vc/)). Git also provides `git commit --fixup` and
+`git rebase --autosquash` which can streamline this process.
+
+# Build and Release Versions
+
+Release versions of Katana are a semantic version with an release candidate tag
+(`rc#`) append for release candidates.
+
+Build/development versions of Katana are in the form:
+`0.1.10rc1.dev+40.0.04ac6f`. This is a release version followed with `.dev` and
```

Comment:
This is a lot to parse at once.

How about:

"... are in the form: <version> .dev + <number> . <number> . <hash>. For example, 0.1.10rc1.dev+40.0.04ac6f is .... "

?